charts/clearblade-iot-enterprise/charts/cb-postgres/templates/postgres-cluster.yaml

```diff
@@ -0,0 +1,97 @@
+{{- if .Values.global.dynamicVolumes }}
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cb-postgres
+  namespace: {{ default "clearblade" .Values.global.namespace }}
+  labels:
+    app: cb-postgres
+  annotations:
+    cnpg.io/skipEmptyWalArchiveCheck: enabled
+spec:
+  # template:
+  #   spec:
+  #     serviceAccountName: {{ .Values.global.gcpGSMServiceAccount }}
+        # tolerations:
+        # - key: node.kubernetes.io/not-ready
+        #   operator: Exists
+        #   effect: NoExecute
+        #   tolerationSeconds: 0
+        # - key: node.kubernetes.io/unreachable
+        #   operator: Exists
+        #   effect: NoExecute
+        #   tolerationSeconds: 0
+  instances: 3
+  bootstrap:
+  {{- if .Values.global.restored }}
+    recovery:
+      source: {{ default "clearblade" .Values.global.folder }}
+  externalClusters:
+    - name: {{ default "clearblade" .Values.global.folder }}
+      barmanObjectStore:
+        destinationPath: "{{ .Values.global.bucket }}"
+        # googleCredentials:
+        #   gkeEnvironment: true
+        googleCredentials:
+          applicationCredentials:
+            name: backup-creds
+            key: gcsCredentials
+        wal:
+          maxParallel: 8
```

Comment:
Docs say that we can setup a backup interval plus retention policy. Do we want to configure that?

[Backup Configuration Docs](https://github.com/cloudnative-pg/charts/blob/main/charts/cluster/README.md#backup-configuration)